### ISSUE TITLE
S2T Example, how make inference on single wav?

### ISSUE THREAD

**Issue by paklau99988** [pytorch/fairseq#3146]
## 📚 Documentation

Hello, 
I need make inference on single wav english speech using your pretrained model. No can find how to do, only see for original data set not single wav.

**Comment by kahne** [comment]
We will make an update shortly to allow using `fairseq-interactive` for this use case.

**Comment by paklau99988** [comment]
Thanks, 
Can work in command line linux machine?

**Comment by paklau99988** [comment]
Any news, please?

**Comment by kahne** [comment]
> Thanks,
> Can work in command line linux machine?

Yes, you should be able to input audio path in CLI and get the text prediction as output on the screen. We will make the update this week. I will notify you then. Thanks for your patience!

**Comment by kahne** [comment]
@paklau99988  We pushed an update for this. Please see the example for usage: https://github.com/pytorch/fairseq/blob/master/examples/speech_to_text/docs/covost_example.md#interactive-decoding

**Comment by guillefix** [comment]
I was trying to get the interactive decoding to work, but I get the following error after inputing my file name

```
/home/guillefix/anaconda3/lib/python3.8/site-packages/torchaudio/backend/utils.py:53: UserWarning: "sox" backend is being deprecated. The default backend will be changed to "sox_io" backend in 0.8.0 and "sox" backend will be removed in 0.9.0. Please migrate to "sox_io" backend. Please refer to https://github.com/pytorch/audio/issues/903 for the detail.
  warnings.warn(
Segmentation fault (core dumped)
```

**Comment by guillefix** [comment]
Ah it turns out that was because my wav had two channels. Now it's returning a result, but it's nonsense.. "all of their has alaiashehead has alaies hae is in theatehousehousehousehas overhas alreashes have juaHahugheaeaeaeheaeaeheaeaeaeaeheaeaeaeheaeaeheaaeaeaeaeheaeaeaeheaaaaeaeaaaeheaeheheaeaaeaaaeheaea"

**Comment by kahne** [comment]
> Ah it turns out that was because my wav had two channels. Now it's returning a result, but it's nonsense.. "all of their has alaiashehead has alaies hae is in theatehousehousehousehas overhas alreashes have juaHahugheaeaeaeheaeaeheaeaeaeaeheaeaeaeheaeaeheaaeaeaeaeheaeaeaeheaaaaeaeaaaeheaeheheaeaaeaaaeheaea"

Can you show me your `config.yaml` file? It's likely speech features were not properly normalized (normalization step defined in `config.yaml`) while the pre-trained model requires normalized features as inputs.

**Comment by rohit10tiwari** [comment]
> > Ah it turns out that was because my wav had two channels. Now it's returning a result, but it's nonsense.. "all of their has alaiashehead has alaies hae is in theatehousehousehousehas overhas alreashes have juaHahugheaeaeaeheaeaeheaeaeaeaeheaeaeaeheaeaeheaaeaeaeaeheaeaeaeheaaaaeaeaaaeheaeheheaeaaeaaaeheaea"
> 
> Can you show me your `config.yaml` file? It's likely speech features were not properly normalized (normalization step defined in `config.yaml`) while the pre-trained model requires normalized features as inputs.

Can you tell me where to find the config.yaml file or how to generate config.yaml for my single .wav file so that i can perform inference via fairseq-interactive ? I am trying to use Covost2 pretrained model provided by you.
